install/Dockerfile.raft

```diff
@@ -0,0 +1,10 @@
+FROM nvcr.io/nvidia/rapidsai/rapidsai-core:23.02-cuda11.8-runtime-ubuntu22.04-py3.10
+
+RUN pip install h5py psutil pyyaml matplotlib
+RUN pip install docker ansicolors
+
+RUN python3 -c 'from pylibraft.neighbors import ivf_pq;'
+
+COPY --from=ann-benchmarks /home/app /home/app
+
+ENTRYPOINT ["python3", "-u", "/home/app/run_algorithm.py"]
```

Comment:
It looks like the final results are still in the container after running the benchmarks. I'm trying to figure out a good way to copy them back onto the local filesystem.

Reply:
I *think* I figured this out. It looks like it works when I set the `WORKDIR` here, though I'm not a docker expert

Reply:
We mount-bind  the host's `results/` as read-write to `/home/app/results`.